models/opnilog/opnilog_parser.py

```diff
@@ -4,6 +4,7 @@
 import time
 
 # Third Party
+import pandas as pd
```

Comment:
Why do we need to import pandas here?

Reply:
the IDE added it automatically as it's used in one of the function in this script.